tests/manage/z_cluster/nodes/test_node_replacement_reactive_aws_ipi.py

```diff
@@ -0,0 +1,146 @@
+import logging
+import pytest
+import random
+from ocs_ci.framework.testlib import (
+    tier4, tier4b, ManageTest, aws_platform_required,
+    ipi_deployment_required, ignore_leftovers)
+from ocs_ci.ocs import machine
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+from tests.helpers import wait_for_resource_state
+from tests.sanity_helpers import Sanity
+from tests.helpers import get_worker_nodes
+from ocs_ci.ocs import ocp
+
+log = logging.getLogger(__name__)
+
+
+@ignore_leftovers
+@tier4
+@tier4b
+@aws_platform_required
+@ipi_deployment_required
+class TestNodeReplacement(ManageTest):
+    """
+    Knip-894 Node replacement - AWS-IPI-Reactive
+    """
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.sanity_helpers = Sanity()
+
+    def test_node_replacement_reactive_aws_ipi(
+            self, nodes, pvc_factory, pod_factory,
+            dc_pod_factory
+    ):
+        """
+        Knip-894 Node replacement - AWS-IPI-Reactive
+
+        """
+        # Get worker nodes
+        initial_nodes = get_worker_nodes()
```

Comment:
please try using nodes.get_typed_nodes()
I think we should use only this function as it is generic for both workers and masters

Reply:
Done